lldb/source/Host/windows/MainLoopWindows.cpp

```diff
@@ -272,4 +272,7 @@ Status MainLoopWindows::Run() {
   return Status();
 }
 
-void MainLoopWindows::Interrupt() { WSASetEvent(m_interrupt_event); }
+bool MainLoopWindows::Interrupt() {
+  WSASetEvent(m_interrupt_event);
```

Comment:
`WSASetEvent` returns a `bool` that indicates if it succeeded or not, so we could return that here.